e2e-tests/insurance/cypress/e2e/journeys/dashboard/dashboard-pagination/dashboard-pagination-15-applications.spec.js

```diff
@@ -11,13 +11,19 @@ const dashboardUrl = `${baseUrl}${DASHBOARD}`;
 
 context(`Insurance - Dashboard - pagination - ${totalApplications} applications`, () => {
   let applications;
+  let abandonedApplication;
 
   before(() => {
     cy.completeSignInAndGoToDashboard().then(({ accountId }) => {
       cy.createApplications(accountId, totalApplications).then((createdApplications) => {
         applications = createdApplications;
       });
 
+      // creates an abandoned application
```

Comment:
I don't think we need this comment, self explanatory

Could we remove the abandoned application from this test and create a new dashboard/pagination test where there are 2 applications, one abandoned, one regular "in progress" application, and then assert the length and status via:

```js
cy.assertLength(table.body.rows(), 1);
dashboardPage.table.body.row(referenceNumber)
```

Should probably add a test for 17 applications, with one of them being abandoned. So the pagination is rendered, but no abandoned applications are present

Then the existing E2E test can remain as a happy path